apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SqlResultStream.cs

```diff
@@ -111,7 +111,12 @@ public async Task<bool> MoveNext(CancellationToken cancellationToken)
         private async Task<PartialResultSet> ComputeNextAsync(CancellationToken cancellationToken)
         {
             // The retry state is local to the method as we're not trying to handle callers retrying.
-            RetryState retryState = new RetryState(_client.Settings.Scheduler ?? SystemScheduler.Instance, _retrySettings);
+            // Consecutive retryable errors will be retried until the call succeeds, or until the call times out or is cancelled.
+            RetryState retryState = new RetryState(_client.Settings.Scheduler ?? SystemScheduler.Instance, _retrySettings, maxConsecutiveErrors: int.MaxValue);
```

Comment:
RetryState is an internal class. If `maxConsecutiveErrors` is not meaningful anymore maybe we can just remove it.

Reply:
Done.